### PR TITLE
Id name validate format

### PR DESCRIPTION
**Pull request by iyeshaya**
## Related Issues
fixes: CIAC-3218

* Added a validation that classifier's id and name are matching. Updated the format of classifier to include update_id too.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
 * Added a new validation to the **validate** command to verify that the docker in use is not deprecated.
 * Added support for multiple ApiModules in the **unify** command
 * Added slack notifier for build failures on the master branch.
+* Added a validation that classifier's id and name are matching. Updated the format of classifier to include update_id too.
```
Comment (dorschw) [inline CHANGELOG.md:7]:
```suggestion
* Added a **validate** check, making sure classifier id and name values match. Updated the classifier **format**   to update the id accordingly.
```

File: demisto_sdk/commands/common/hook_validations/classifier.py [hunk 2]
```diff
@@ -43,7 +43,8 @@ def is_valid_classifier(self, validate_rn=True, id_set_file=None, is_circle=Fals
                 self.is_valid_to_version(),
                 self.is_to_version_higher_from_version(),
                 self.is_valid_type(),
-                self.is_incident_field_exist(id_set_file, is_circle)
+                self.is_incident_field_exist(id_set_file, is_circle),
+                self.is_id_equals_name()
```
Comment (dorschw) [inline demisto_sdk/commands/common/hook_validations/classifier.py:47]:
```suggestion
                self.is_id_equals_name(),
```

File: demisto_sdk/commands/common/hook_validations/classifier.py [hunk 3]
```diff
@@ -52,9 +53,18 @@ def is_valid_classifier(self, validate_rn=True, id_set_file=None, is_circle=Fals
             self.is_valid_from_version(),
             self.is_valid_to_version(),
             self.is_to_version_higher_from_version(),
-            self.is_valid_type()
+            self.is_valid_type(),
+            self.is_id_equals_name()
```
Comment (dorschw) [inline demisto_sdk/commands/common/hook_validations/classifier.py:57]:
```suggestion
            self.is_id_equals_name(),
```

**Review by dorschw — COMMENTED** [review]
Nice
Do we have any content types that do not have it, except for `classifier`?

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,7 @@
 * Added a new validation to the **validate** command to verify that the docker in use is not deprecated.
 * Added support for multiple ApiModules in the **unify** command
 * Added slack notifier for build failures on the master branch.
+* Added a **validate** check, making sure classifier id and name values match. Updated the classifier **format**   to update the id accordingly.
```
Comment (dorschw) [inline CHANGELOG.md:7]:
```suggestion
* Added a **validate** check, making sure classifier id and name values match. Updated the classifier **format** to update the id accordingly.
```

**Review by dorschw — APPROVED** [review]

**Comment by ShahafBenYakir** [comment]
making sure our format command fixes this validation error

**Pull request merged**